config/install/filter.format.stanford_html.yml

```diff
@@ -2,11 +2,15 @@ langcode: en
 status: true
 dependencies:
   config:
-    - core.entity_view_mode.media.full
+    - core.entity_view_mode.media.stanford_image_large
+    - core.entity_view_mode.media.stanford_image_medium
+    - core.entity_view_mode.media.stanford_image_stanford_circle
```

Comment:
These configs aren't part of this module or stanford_media. they should be removed.